File: questions/Jewels_and_Stones.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        jewel = list(J)
        count = 0 
        for x in list(S):
            if x in jewel:
                count +=1 
        return count 

File: questions/Jewels_and_Stones.go
/*借鉴了讨论中一个非常巧妙的方法，见readme*/
func numJewelsInStones(J string, S string) int {
    Jewel := make(map[rune]int)
    for _, word := range J{
        Jewel[word] = 1
    }
    count := 0 
    for _, letter  := range S{
         count += Jewel[letter]
    }
    return count 
}
